bin/load_exp_design.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -e
+
+scriptDir=$(cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+source $scriptDir/db_scxa_common.sh
+
+dbConnection=${dbConnection:-$1}
+condensed_sdrf_file=${CONDENSED_SDRF_FILE:-$2}
+sdrf_file=${SDRF_FILE:-$3}
+
+# Check that necessary environment variables are defined.
+[ -z ${dbConnection+x} ] && echo "Env var dbConnection for the database connection needs to be defined. This includes the database name." && exit 1
+[ -z ${CONDENSED_SDRF_FILE+x} ] && echo "Env var CONDENSED_SDRF_FILE for the experiment design data needs to be defined." && exit 1
+[ -z ${SDRF_FILE+x} ] && echo "Env var SDRF_FILE for column sequence of experiment design needs to be defined." && exit 1
```

Comment:
If these checks are OK, then they should be above L8.

Or are you intending to check here `condensed_sdrf_file` and `sdrf_file`?

Reply:
I followed the convention data production team has been using in the scripts. For example, [here](https://github.com/ebi-gene-expression-group/db-scxa/blob/develop/bin/add_exps_to_collection.sh) and [here](https://github.com/ebi-gene-expression-group/db-scxa/blob/develop/bin/load_db_scxa_analytics.sh).

I do not have a strong opinion about this was just following the format found in other scripts.